docs/scripting/language/operator.md

```diff
@@ -0,0 +1,130 @@
+---
+id: Directives
```

Comment:
"Operators" and the capitalisation of the filename should match the other files.